mkrelease.sh

```diff
@@ -82,21 +77,9 @@ chmod 444 $2
 }
 inst $tmpdir/${name}.tar.gz	$dst/${name}.tar.gz
 inst $tmpdir/${name}.tar.bz2	$dst/${name}.tar.bz2
-inst $tmpdir/CONTRIBUTORS	$dst/CONTRIBUTORS.txt
-inst $tmpdir/COPYING		$dst/COPYING.txt
-inst $tmpdir/README		$dst/README.txt
-inst $tmpdir/CREDITS		$dst/CREDITS.txt
-inst $tmpdir/SPONSORS		$dst/SPONSORS.txt
 inst $tmpdir/ChangeLog		$dst/ChangeLog.txt
```

Comment:
I assume something on the web server still needs this installation/renaming. If possible, please remove that something (for newer releases) and just post a link to the right GitHub file, dropping this customization line. As a positive side effect, this will avoid the broken code that currently presents ChangeLog contents as HTML (search for `Make %` in the current version of http://www.squid-cache.org/Versions/v6/ChangeLog.html to see that bug).

Reply:
Thanks for mentioning the display issue, it is purely a web PHP typo/bug. I will look into it shortly.

Slow down :-) I am working on these changes methodically. This request and many other updates will be following, but not in scope of this PR.